Userland/Libraries/LibWeb/CSS/FontFaceSet.h

```diff
@@ -50,7 +50,11 @@ class FontFaceSet final : public DOM::EventTarget {
     virtual void visit_edges(Cell::Visitor&) override;
 
     JS::NonnullGCPtr<JS::Set> m_set_entries;
-    JS::GCPtr<WebIDL::Promise> m_ready_promise; // [[ReadyPromise]]
+    JS::GCPtr<WebIDL::Promise> m_ready_promise;      // [[ReadyPromise]]
+                                                     //
+    Vector<JS::Handle<FontFace>> m_loading_fonts {}; // [[LoadingFonts]]
+    Vector<JS::Handle<FontFace>> m_loaded_fonts {};  // [[LoadedFonts]]
+    Vector<JS::Handle<FontFace>> m_failed_fonts {};  // [[FailedFonts]]
```

Comment:
I'm skeptical that we need to hold onto a vector of handles. if we visit every element in each list in visit_edges, it should be fine to have a `Vector<JS::NonnullGCPtr<FontFace>>`

Reply:
Okay I'll make that change. Thanks.